tests/common/test_optogenetics.py

```diff
@@ -0,0 +1,100 @@
+from ndx_optogenetics import OptogeneticVirusInjection
+
+
+def test_optogenetics(
```

Comment:
This seems separable into multiple tests: `test_opto_injection`, `test_opto_fiber`, ...